mc-sema/cfgToLLVM/x86Instrs_MOV.cpp

```diff
@@ -573,6 +573,14 @@ static InstTransResult translate_MOVao (NativeModulePtr natM, BasicBlock *& bloc
     Function *F = block->getParent();
     Module *M = F->getParent();
 
+    // this is awful, but sometimes IDA detects the immediate
+    // as a memory reference. However, this instruction can only
+    // have an immediate, so this is safe
+    if( ip->has_imm_reference || ip->has_mem_reference ) {
```

Comment:
For this specific instruction there should only be an immediate operand and it should always be a reference to something.